package.json

```diff
@@ -25,6 +25,7 @@
     "express": "^4.16.3",
     "express-validator": "^5.2.0",
     "http-status-codes": "^1.3.0",
+    "lodash": "^4.17.10",
```

Comment:
We could use https://facebook.github.io/immutable-js/docs/#/ instead of lodash. There is also a task in the integreat-webapp which wants to do this. We could evaluate this here.

Reply:
I will do this in another task.